docs/en/ingest-management/release-notes/release-notes-8.9.asciidoc

```diff
@@ -0,0 +1,233 @@
+// Use these for links to issue and pulls.
+:kibana-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:beats-issue: https://github.com/elastic/beats/issues/
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.9.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.9.0 relnotes
+
+[[release-notes-8.9.0]]
+== {fleet} and {agent} 8.9.0
+
+Review important information about the {fleet} and {agent} 8.8.0 release.
+
+[discrete]
+[[security-updates-8.9.0]]
+=== Security updates
+
+{fleet-server}::
+* Use a verified base image for building Fleet Server binaries. {fleet-server-pull}2339[#2339]
+
+[discrete]
+[[breaking-changes-8.9.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+[discrete]
+[[breaking-2890]]
+.Status command has been changed.
+[%collapsible]
+====
+*Details* +
+The {agent} `status` command has been changed so that the default human output now uses a list format and summaries output.
+
+*Impact* +
+Full human output can be obtained with the new `full` option.
+For for information, refer to {agent-pull}2890[#2890].
+====
+
+[discrete]
+[[breaking-2531]]
+.API default error code is now 500.
+[%collapsible]
+====
+*Details* +
+Previously, when {fleet-server} encountered an unexpected error it resulted in a `Bad Request` response.
+
+*Impact* +
+Now, any unexpected error returns an `Internal Server Error` response while keeping most of the current behavior
+unchanged. On expected failure paths (for example, Agent Inactive, Missing Agent ID, Missing Auth Header) a `Bad Request` response is returned. For more information, refer to {fleet-server-pull}2531[#2531].
+====
+
+[discrete]
+[[new-features-8.9.0]]
+=== New features
+
+The 8.9.0 release Added the following new and notable features.
+
+{fleet}::
+* Adds CloudFormation install method to CSPM. {kibana-pull}159994[#159994]
+* Adds flags to give permissions to write to any dataset and namespace. {kibana-pull}157897[#157897]
+* Disables Agent ID verification for Observability projects. {kibana-pull}157400[#157400]
+* Setup `ignore_malformed` in {fleet}. {kibana-pull}157184[#157184]
```

Comment:
@gsantoro Do you want these features to show up in Fleet release notes?

Reply:
@juliaElastic I believe Giuseppe is currently on leave.